### PR TITLE
Add desktop version style

### PR DESCRIPTION
**Pull request by Ghilain**
Add new css file for desktop version deskstyle.css
Add link menu for desktop in index.html page
Fix style.css file for mobile version
add new folder images contain all image for desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -147,25 +175,30 @@ <h2 class="titlpost">
           </div>
           <div class="cardwork6">
             <div class="container">
-              <div class="action">
-                <button class="buton1" type="button"> See Project</button>
+              <div class="action action3">
+                <button class="buton1 buton4" type="button"> See Project</button>
+              </div>
+              <div class="backimage">
+                <div class="cursor"></div>
```
Comment (Amrhub) [inline index.html:182]: Great implementation of UI design, we are just missing this tiny requirement let's kindly implement it.

- [x]  I know it's kind of confusing but if you pay closer attention in Figma design project-1 is like the rest of the projects but it has a hover-effect where when you hover over it the dark-background color is removed and the title, description, and technologies are also removed
then you are left with `See Project`

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @Ghilain,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Ghilain) [reply index.html:182]: Thank you @Amrhub

**Review by Gambit142 — APPROVED** [review]
Hi @Ghilain 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**